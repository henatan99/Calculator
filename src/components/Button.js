/* eslint-disable react/prop-types */ // TODO: upgrade to latest eslint tooling
const Button = ({name}) => <button className="button" type="submit">{name}</button>;

export default Button;
