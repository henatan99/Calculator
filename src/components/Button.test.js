import React from 'react';
import { create } from 'react-test-renderer';
import Button from './Button';

it('renders correctly', () => {
    const tree = create(<Button />).toJSON;
    expect(tree).toMatchSnapshot();
})

it('renders correctly', () => {
    const tree = create(<Button name="1" />).toJSON;
    expect(tree).toMatchSnapshot();
})

it('renders correctly', () => {
    const tree = create(<Button color="red" wide="100px" />).toJSON;
    expect(tree).toMatchSnapshot();
})