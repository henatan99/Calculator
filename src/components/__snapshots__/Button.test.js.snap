// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `[Function]`;

exports[`renders correctly 2`] = `[Function]`;

exports[`renders correctly 3`] = `[Function]`;
